source/Octopus.Server.Client/Serialization/TriggerFilterConverter.cs

```diff
@@ -17,7 +17,8 @@ public class TriggerFilterConverter : InheritedClassConverter<TriggerFilterResou
               { TriggerFilterType.DaysPerMonthSchedule, typeof (DaysPerMonthScheduledTriggerFilterResource)},
               { TriggerFilterType.DaysPerWeekSchedule, typeof (DaysPerWeekScheduledTriggerFilterResource)},
               { TriggerFilterType.CronExpressionSchedule, typeof (CronScheduledTriggerFilterResource)},
-              { TriggerFilterType.FeedFilter, typeof (FeedFilterResource)}
+              { TriggerFilterType.FeedFilter, typeof (FeedFilterResource)},
+              { TriggerFilterType.Arc, typeof (ArcFeedFilterResource)}
```

Comment:
given everyone else has filter in the name - probably should be TriggerFilterType.ArcFilter

Reply:
Good point, thanks